src/v/kafka/server/protocol.h

```diff
@@ -99,6 +134,7 @@ class protocol final : public rpc::server::protocol {
     ss::sharded<security::credential_store>& _credentials;
     ss::sharded<security::authorizer>& _authorizer;
     ss::sharded<cluster::security_frontend>& _security_frontend;
+    qdc_monitor _qdc;
```

Comment:
very very cool! rename to `_qdcm` ? m for monitor?

Reply:
fixed